FEM/pcs_dm.cpp

```diff
@@ -1287,7 +1289,7 @@ void CRFProcessDeformation::CreateInitialState4Excavation()
 	UpdateInitialStress(false); // s-->s0
 	m_msh->ConnectedElements2Node();
 	for (i = 0; i < m_msh->GetNodesNumber(false); i++)
-		for (j = 0; j < NS + 1; j++)
+		for (j = 0; j < NS + 2; j++)
```

Comment:
You can declare a variable for NS +1 or NS +2 (if DILATANCY ) outside of   the  loop over nodes.